CHANGES.md

```diff
@@ -21,6 +21,7 @@ Change Log
 * Added `terrainProviderChanged` event to `Scene` and `Globe`
 * Added `CullingVolume.fromBoundingSphere`.
 * Added `debugShowShadowVolume` to `GroundPrimitive`.
+* Fixed issue where Matrix4.fromCamera was taking eye/target instead of position/direction. [3927](https://github.com/AnalyticalGraphicsInc/cesium/issues/3927)
```

Comment:
Small issue I just noticed, Matrix4.fromCamera should be surrounded by `` like the other items for markdown styling purposes.